ouroboros-consensus/src/Ouroboros/Consensus/Node/DbLock.hs

```diff
@@ -0,0 +1,67 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Ouroboros.Consensus.Node.DbLock (
+    DbLocked (..)
+  , dbLockFile
+  , lockDB
+  ) where
+
+import           Control.Concurrent
+import           Control.Monad (void)
+import           Data.Text (Text)
+import           Ouroboros.Consensus.Storage.FS.API
```

Comment:
Add a blank space above this line to separate imports from this package from the others.